src/GeneratedHydrator/CodeGenerator/Visitor/HydratorMethodsVisitor.php

```diff
@@ -193,3 +212,53 @@ static function (ClassMethod $method) use ($name) : bool {
         return $method;
     }
 }
+
+/**
+ * @internal
+ */
+final class ObjectProperty
```

Comment:
Let's please split this to a separate file

Reply:
If you wish so.